packs/data/feats.db/perfect-resistance.json

```diff
@@ -25,7 +25,35 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.houseOfPerfection.element",
+                "priority": 1,
+                "value": ""
```

Comment:
empty string?

Reply:
It's there to be a default in the case where the actor has an old Student of Perfection Dedication, or where they add Perfect Resistance without having Student of Perfection Dedication first.

With the empty string, the console emits:
main.bundle.js:1 PF2e System | Resistance rules element on item Perfect Resistance (Actor.GSydcBd6EK220YLZ.Item.5vpIsOm4yGR4MhcR) failed to validate: Type "" is unrecognized

Without the ActiveEffectLike RE, the console emits:
main.bundle.js:1 PF2e System | Resistance rules element on item Perfect Resistance (Actor.GSydcBd6EK220YLZ.Item.ljEEotOWNllPKeCY) failed to validate: Failed to resolve injected property
main.bundle.js:1 PF2e System | Resistance rules element on item Perfect Resistance (Actor.GSydcBd6EK220YLZ.Item.ljEEotOWNllPKeCY) failed to validate: Type "undefined" is unrecognized

Since it's just warnings in both cases, we can just remove this if preferred. It's a holdover from the initial implementation that @TikaelSol helped with.

Alternately, we could default to a sensible value like "fire", but then users might be more confused.

Reply:
If it's going to fail, it can just fail. Two warnings isn't a big deal.